exploration/estimate_real_user_pp/error_calc.ipynb

```diff
@@ -0,0 +1,360 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Historical pp curve error calculation: Euclidean distance MSE"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append('../..')\n",
+    "from pymongo import UpdateOne\n",
+    "from pymongo import MongoClient\n",
+    "from tqdm import tqdm\n",
+    "import numpy as np\n",
+    "from exploration.config import mongo_inst\n",
+    "from mlpp.data_collection.sample import osuDumpSampler\n",
+    "import datetime\n",
+    "from datetime import datetime\n",
+    "import pprint\n",
+    "import matplotlib.pyplot as plt\n",
+    "# from fastdtw import fastdtw\n",
+    "# from scipy.spatial.distance import euclidean"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = MongoClient(port=27017)\n",
+    "db = client.osu_top_db\n",
+    "top_db = mongo_inst[\"osu_top_db\"]\n",
+    "test = osuDumpSampler(top_db)\n",
+    "user_ids = list(map(lambda c: c['_id'], top_db['osu_user_stats'].find({}, {})))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Find users that have more than 50 data points & creates a new list of user ids\n",
+    "big_user_ids = []\n",
+    "subset_user_ids = user_ids[0:100] # the first ten had 5, the second had 6, the third had 5\n",
+    "\n",
+    "for i in subset_user_ids:\n",
+    "    datapts = len(list(db.osu_track_updates.find({\"user_id\": i}, {\"date\":1})))\n",
+    "    if datapts > 50: #arbitrary number\n",
+    "        big_user_ids.append(i)\n"
```

Comment:
You could use aggregation here: group by user_id -> sort by count -> get all ids with count > 50